ui/src/app/shared/service/defaulttypes.ts

```diff
@@ -91,6 +93,61 @@ export module DefaultTypes {
 
   export enum PeriodString { DAY = 'day', WEEK = 'week', MONTH = 'month', YEAR = 'year', CUSTOM = 'custom' };
 
+  export namespace History {
+
+    export enum YAxisTitle {
+      PERCENTAGE,
+      ENERGY
+    }
+    export type InputChannel = {
+
+      /** Must be unique, is used as identifier in {@link ChartData.input} */
+      name: string,
+      powerChannel: ChannelAddress,
+      energyChannel?: ChannelAddress
+
+      /** Choose between predefined converters */
+      converter?: (value: number) => number | null,
+    }
+    export type DisplayValues = {
+      name: string,
+      /** suffix to the name */
+      nameSuffix?: (energyValues: QueryHistoricTimeseriesEnergyResponse) => number | string,
+      /** Convert the values to be displayed in Chart */
+      converter: () => number[],
+      /** If dataset should be hidden on Init */
+      hiddenOnInit?: boolean,
+      /** default: true, stroke through label for hidden dataset */
+      noStrokeThroughLegendIfHidden?: boolean,
+      /** color in rgb-Format */
+      color: string,
+      /** the stack for barChart */
+      stack?: number,
+    }
+    /**
+   * Data from a subscription to Channel or from a historic data query.
+   * 
+   * TODO Lukas refactor

```

Comment:
🤔

Reply:
im not sure, why we added this comment, i removed it